DataManagementSystem/DB/FileCatalogDB.py

```diff
@@ -1021,14 +1021,21 @@ def rebuildDirectoryUsage(self):
     result = self.dtree._rebuildDirectoryUsage()
     return result
 
-  def repairCatalog(self, directoryFlag=True, credDict={}):
+  def repairCatalog(self, credDict={}):
     """ Repair catalog inconsistencies
     """
-    result = S_OK()
-    if directoryFlag:
-      result = self.dtree.recoverOrphanDirectories(credDict)
 
-    return result
+    result = self.securityManager.hasAdminAccess(credDict)
```

Comment:
use `self._checkAdminPermission` ? On the other hand t his method is quite useless, we could remove it from everywhere...